cc-test/tests/output.rs

```diff
@@ -0,0 +1,68 @@
+use std::fs;
+use std::path::PathBuf;
+
+#[test]
+fn cargo_warnings_on() {
+    if env!("TEST_WARNINGS_ON") == "0" {
```

Comment:
We already skip `run_forked_capture_output(&out, "warnings-on");` if `TEST_WARNINGS_ON` is ser to 0, do we still need this?

Reply:
I guess we could use the presence of the build artifacts (i.e. the stdout and stderr file) to detect whether to run the test, but then that could also be caused by someone modifying the build script and forgetting to update it here. Since the test would be silently skipped in that case, I would argue that we should keep it as explicit as possible.